src/instruments/ecam-page-template/template.js

```diff
@@ -36,36 +36,40 @@
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
-/* global BaseInstrument */
-/* global registerInstrument */
+/* global Airliners */
 
-// eslint-disable-next-line camelcase
-class A32NX_PAGE_NAME_Logic extends Airliners.EICASTemplateElement {
-    get templateID() {
-        return 'A32NX_PAGE_NAME_TEMPLATE';
-    }
+let A32NX_PAGE_NAME;
```

Comment:
Isn't this a dangerous name to use as part of the template.js? As far as I understand the string `PAGE_NAME` will be replaced by `door` through our SD rollup.js, so probably any kind of variable should stay well clear of that string?

Reply:
It ends up being `A32NX_door_page` which I think is reasonably safe.